content/zh/docs/installing-on-kubernetes/faq/configure-a-booster-for-installation/为安装配置加速器.md

```diff
@@ -0,0 +1,90 @@
+---
```

Comment:
This is not the correct location of the file. The file should be in docs/installing-on-linux/faq/configure-booster/